modules/instrument_list/locale/instrument_list.pot

```diff
@@ -21,3 +21,50 @@ msgstr ""
 msgid "Instrument List"
 msgstr ""
 
+msgid "Instruments"
+msgstr ""
+
+msgid "Visit to Site"
+msgstr ""
+
+msgid "Within Optimal"
+msgstr ""
+
+msgid "Within Permitted"
+msgstr ""
+
+msgid "Behavioural Battery of Instruments"
+msgstr ""
+
+msgid "Data Entry"
+msgstr ""
+
+msgid "Administration"
+msgstr ""
```

Comment:
Data Entry, Administration, and Double Data Entry should probably be in the loris.po file? They're common LORIS terms.